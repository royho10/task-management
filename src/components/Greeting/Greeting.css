.greeting {
	margin-top: 60px;
	height: auto;
	display: block;
	position: fixed;
	width: 100%;
	text-align: center;
}