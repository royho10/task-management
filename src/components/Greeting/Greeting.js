import React from 'react';

const Greeting = ({ name }) => {
	return (
		<div className='center f3 mv2'>
			{`Hello ${name}, nice to have you back :)`}
		</div>
	);
}

export default Greeting;