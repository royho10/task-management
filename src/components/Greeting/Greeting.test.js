import { shallow } from 'enzyme';
import React from 'react';
import Greeting from './Greeting';

it('expect to render Greeting component', () => {
	expect(shallow(<Greeting />)).toMatchSnapshot();
})