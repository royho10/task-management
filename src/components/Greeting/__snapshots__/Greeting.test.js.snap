// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Greeting component 1`] = `
<div
  className="f3 mv2"
>
  Hello undefined, nice to have you back :)
</div>
`;
